index.html

```diff
@@ -106,6 +106,11 @@ <h2 id="downloads">Installation</h2>
     <td>pkg_add</td>
     <td><a href="https://www.openbsd.org/">OpenBSD</a></td>
     <td><kbd>pkg_add yash</kbd></td>
+  </tr>
+   <tr>
+    <td>Portage</td>
+    <td><a href="https://www.gentoo.org/">Gentoo</a></td>
+    <td><kbd>emerge --ask app-shells/yash</kbd></td>
```

Comment:
Why `--ask` here? We don't have similar options in the other examples.

Reply:
Gentoo is a source based distribution so most Gentoo users would like to see what will be Installed beforehand and also possibly [tweak any USE flags](https://wiki.gentoo.org/wiki/USE_flag).